scripts/renew.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -x
+
+service nginx stop
+
+docker run -ti --rm \
+    -v /var/log/letsencrypt:/var/log/letsencrypt \
+    -v $(pwd)/credentials:/root/.secrets \
+    -v $(pwd)/letsencrypt:/etc/letsencrypt \
+    -v $(pwd)/nginx:/etc/nginx/conf.d \
+    certbot/dns-aliyun \
+    renew -q
+
+service nginx start
```

Comment:
`nginx reload` will be more safer

Reply:
You are right. I have improved it by checking the service status first.